script/base/DeployMorphoTokenBase.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../../lib/forge-std/src/Script.sol";
+import "../../lib/forge-std/src/console.sol";
+
+import {MorphoTokenOptimism} from "../../src/MorphoTokenOptimism.sol";
+import {ERC1967Proxy} from
+    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployMorphoTokenBase is Script {
+    address public constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+    address public REMOTE_TOKEN;
+    address public constant BRIDGE = 0x4200000000000000000000000000000000000010;
+
+    bytes32 public IMPLEMENTATION_SALT;
+    bytes32 public PROXY_SALT;
```

Comment:
Let's open an issue to remind to set them, or say that we leave them explicitly at zero.